language/src/rustspec_to_coq.rs

```diff
@@ -790,6 +790,10 @@ fn translate_func_name<'a>(
 
 fn translate_expression<'a>(e: Expression, top_ctx: &'a TopLevelContext) -> RcDoc<'a, ()> {
     match e {
+        Expression::MonadicLet(..) => panic!("TODO: Coq support for Expression::MonadicLet"),
```

Comment:
@cmester0 do you want to implement this?

Reply:
I can try and fill this in, should just map to some existing notation in Coq, doing the monadic binding.